src/_includes/shipping-status-note.md

```diff
@@ -1,3 +1,3 @@
 
 {:.bs-callout-warning}
-Due to the impending shutdown of Temando, the provider of the technology behind Magento Shipping, it is no longer possible to create a new Magento Shipping account. Support for current  Magento Shipping deployments for all existing customers will continue. For detailed status information and recommendations for new shipping implementations in Magento, see our [product information](https://magento.com/shipping){:target="_blank"}  page.
+Due to the impending shutdown of Temando, the provider of the technology behind Magento Shipping, it is no longer possible to create a new Magento Shipping account. Support for current Magento Shipping deployments for all existing customers will continue. For detailed status information and recommendations for new shipping implementations in Magento, see our [product information](https://magento.com/shipping){:target="_blank"}  page.
```

Comment:
```suggestion
Due to the impending shutdown of Temando, the provider of the technology behind Magento Shipping, you can no longer create a new Magento Shipping account. Support for current Magento Shipping customers will continue. For detailed status information and recommendations for new shipping implementations in Magento, see our [product information](https://magento.com/shipping){:target="_blank"}  page.
```

Reply:
This is from Product Management and wording is in line with their communications. They will be editing going forward. So leaving this for now.